tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/testcontainer/HelmChartContainer.java

```diff
@@ -433,7 +437,8 @@ private void addHiveMQHelmRepo() {
     }
 
     private static void createContainerRegistrySecret(
-            final @NotNull String namespace, final @NotNull K3sContainer container) {
+            final @NotNull String namespace,
```

Comment:
That's still this formatter bug: https://youtrack.jetbrains.com/issue/IDEA-295041/Java-code-formatting-with-chop-down-on-Method-call-arguments-is-not-stable

I just like to restore the "correct" formatting sometimes, since that's the intended one, if the formatter would be stable.